### PR TITLE
feat(condo): DOMA-417 add payment orders to payments

### PR DESCRIPTION
**Pull request by DanilovMaxim13**
Before: 
image

After:
image

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts [hunk 1]
```diff
@@ -37,6 +38,7 @@ export function usePaymentsTableFilters (
     const typeMessage = intl.formatMessage({ id: 'pages.condo.payments.type' })
     const enterTypeMessage = intl.formatMessage({ id: 'pages.condo.payments.enterType' })
     const statusTitle = intl.formatMessage({ id: 'Status' })
+    const paymentOrderTitle = intl.formatMessage({ id: 'PaymentOrder' })
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts:41]: We use `CamelCaseNames` for translation consts

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts [hunk 2]
```diff
@@ -113,6 +115,17 @@ export function usePaymentsTableFilters (
                     },
                 },
             },
+            {
+                keyword: 'order',
+                filters: [orderFilter],
+                component: {
+                    type: ComponentType.Input,
+                    modalFilterComponentWrapper: {
+                        label: paymentOrderTitle,
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts:124]: How about fix useMemo deps?)
Seems like it missing new translation as dep (and other stuff as well)

File: apps/condo/lang/ru/ru.json [hunk 3]
```diff
@@ -1268,6 +1269,7 @@
   "excelExport.headers.payments.type": "Тип",
   "excelExport.headers.payments.transaction": "Транзакция",
   "excelExport.headers.payments.status": "Статус",
+  "excelExport.headers.payments.order": "Платежное поручение",
```
Comment (dkoviazin) [inline apps/condo/lang/ru/ru.json:1272]: Better to use some short label
**ПП** or **П/П**
as the number is usually small - like 
**619284**

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts (hunk 1 above)
Comment (DanilovMaxim13) [reply apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts:41]: I use CamelCaseNames. Is not it so?

File: apps/condo/lang/ru/ru.json (hunk 3 above)
Comment (DanilovMaxim13) [reply apps/condo/lang/ru/ru.json:1272]: Will we lose readability?

Comment (dkoviazin) [reply apps/condo/lang/ru/ru.json:1272]: No - but can be helpful if you need to make column width less

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts:41]: **U**pperCamelCase

File: apps/condo/lang/ru/ru.json (hunk 3 above)
Comment (DanilovMaxim13) [reply apps/condo/lang/ru/ru.json]: Done. Add tooltip for short name column.

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts (hunk 1 above)
Comment (DanilovMaxim13) [reply apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts]: Update const in file

File: apps/condo/domains/acquiring/hooks/usePaymentsTableColumns.ts [hunk 4]
```diff
@@ -97,9 +106,5 @@ export function usePaymentsTableColumns (currencyCode: string, openStatusDescMod
         }
 
         return Object.values(columns)
-    }, [
-        filters,
-        sorterMap,
-        currencyCode,
-    ])
+    }, [filters, DateTitle, intl, AccountTitle, AddressTitle, UnitNameTitle, TypeTitle, TransactionTitle, StatusTitle, openStatusDescModal, PaymentOrderColumnTitle, PaymentOrderTooltipTitle, PaymentAmountTitle, currencyCode])
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/hooks/usePaymentsTableColumns.ts:109]: Horizontal scroll is not preferred :)

File: apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts [hunk 5]
```diff
@@ -122,14 +134,15 @@ export function usePaymentsTableFilters (
...
-                        label: statusTitle,
-                        size: FilterComponentSize.Large,
+                        label: StatusTitle,
+                        size: FilterComponentSize.Medium,
                     },
                 },
             },
         ]
-    }, [get(billingContext, 'id', null), organizationId])
+    }, [AccountTitle, AddressMessage, DateMessage, EndDateMessage, EnterAddressMessage, EnterStatusMessage, EnterTypeMessage, PaymentOrderTitle, StartDateMessage, StatusTitle, TypeMessage, billingContext, organizationId, statusOptions])
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts:146]: Use 
```
[
    ...,
    ...,
]
```

if line is too long

**Review by SavelevMatthew — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**